src/libs/CardUtils.ts

```diff
@@ -194,6 +195,16 @@ function getCardFeedIcon(cardFeed: string): IconAsset {
     return Illustrations.AmexCompanyCards;
 }
 
+function getDescriptionForPolicyDomainCard(domainName: string) {
+    const match = domainName.match(CONST.REGEX.EXPENSIFY_POLICY_DOMAIN_NAME);
```

Comment:
So this would be more like `isWorkspaceFeed` if the domain name matches the `expensify-policy<policyID>.exfy `format